ccdaservice/serveccda.js

```diff
@@ -2346,7 +2389,9 @@ function populateHeader(pd) {
         },
         "title": name,
         "date_time": {
-            "date": fDate(pd.created_time_timezone),
+            // php already sends this in YYYYMMDDHHmmss(+/-)zzzz format
+            // so we can just take it straight away for the document head
+            "date": pd.created_time_timezone,
```

Comment:
@sjpadgett You may want to look at this closely.  I thought this was all working but I must not have reset my node server when I tested with ETT originally.  The fDate formatting was changing the timezone to an invalid format that failed the ETT validation (it wants the timestamps to be without any spaces or hyphens).